bib/internal/util/util.go

```diff
@@ -26,3 +26,48 @@ func RunCmdSync(cmdName string, args ...string) error {
 	}
 	return nil
 }
+
+func checkContainerImageExists(storageRef string) (bool, error) {
+	cmd := exec.Command("skopeo", "inspect", storageRef)
```

Comment:
An important best practice with `skopeo inspect` is to use `-n`; otherwise it may enumerate tags in the remote registry which is *expensive* in the general case when dealing with remote registries.

I don't think it matters much here, but just noting.

May help to pass e.g. `-f {{.Digest}}` or something so skopeo knows we don't need much information.

Reply:
Thanks that's great feedback, will fix that up